sdk/servicebus/Microsoft.Azure.ServiceBus/src/SessionHandlerOptions.cs

```diff
@@ -84,7 +84,7 @@ public int MaxConcurrentSessions
             }
         }
 
-        /// <summary>Gets or sets whether the autocomplete option of the session handler is enabled.</summary>
+        /// <summary>Gets or sets whether the autocomplete option for messages in the session handler is enabled.</summary>
```

Comment:
>autocomplete [](start = 46, length = 12)

Given that we are trying to fix the document, lets make it clear that  if the handler returns without any failure, then the message is completed, and if any exception is thrown, the message is abandoned.